packages/react-router-dom/__tests__/nav-link-active-test.tsx

```diff
@@ -333,11 +333,7 @@ describe("NavLink using a data router", () => {
   it("applies the default 'active'/'pending' classNames to the underlying <a>", async () => {
     let deferred = defer();
     render(
-      <DataBrowserRouter
-        window={getWindow("/foo")}
-        hydrationData={{}}
-        fallbackElement={<span />}
```

Comment:
Now that `fallbackElement` is optional we don't need these props in all of our tests